src/resources/Customer.js

```diff
@@ -189,6 +193,23 @@ class Customer extends Resource {
     return this.resource(Message, { code: this.code, ...payload });
   }
 
+  /**
+   * Gets a context for querying system-wide roles
+   * @returns {RolesContext} Context for querying system-wide roles
+   */
+  roles() {
```

Comment:
This should be moved up to `src/resources/index.js` because it's not actually relative to the customer.